apps/dashboard/app/helpers/dashboard_helper.rb

```diff
@@ -6,11 +6,19 @@ def markdown(text)
     text
   end
 
-  def logo_image_tag(url)
+  def logo_image_tag(url, responsive = false)
```

Comment:
This is control coupling. The new addition adds enough bulk to the method that it would be cleaner to  just be a separate method.

```
def responsive_logo_image_tag(url)
  ...
end

def logo_image_tag(url)
  ...
end
```

Reply:
Though I'm inclined to suggest - if svg, specify a maximum height pulled from `Configuration.logo_max_height`